recipes/opengv/all/patches/0003-let-conan-handle-shared-fpic-and-marchs.patch

```diff
@@ -11,3 +11,14 @@
  ELSE()
    OPTION(BUILD_SHARED_LIBS "Build shared libraries" OFF)
    OPTION(BUILD_POSITION_INDEPENDENT_CODE "Build position independent code (-fPIC)" ON)
+ ENDIF()
+ 
+-IF(MSVC)
+-  add_compile_options(/wd4514 /wd4267 /bigobj)
+-  add_definitions(-D_USE_MATH_DEFINES)
++IF(1)
```

Comment:
This patch has been updated to also disable the hardcoding of the `march` flags so we can handle those through conan